### PR TITLE
Remove v1 jpv1 from operator UI

### PR DESCRIPTION
**Pull request by jkongie**

### CODE REVIEW

File: core/services/job/orm.go [hunk 1]
```diff
@@ -403,6 +403,7 @@ func (o *orm) JobsV2() ([]Job, error) {
 	var jobs []Job
 	err := postgres.GormTransactionWithDefaultContext(o.db, func(tx *gorm.DB) error {
 		err := PreloadAllJobTypes(tx).
+			Preload("JobSpecErrors").
```
Comment (archseer) [inline core/services/job/orm.go:406]: I have a similar change here (together with some more API changes): https://github.com/smartcontractkit/chainlink/pull/4715/commits/a93a798f47f6cf76873e67b55cafe864bb78fe09#diff-c15a74a1728815973c97643627390591c30f41dc0501779bc30e664ffa137fe5R417

FindJob also needed the preload.

**Comment by archseer** [comment]
Should this PR maybe target #4718 instead of develop?

**Review by samsondav — CHANGES_REQUESTED** [review]
Can we strip out the failing integration tests

File: core/services/job/orm.go (hunk 1 above)
Comment (jkongie) [reply core/services/job/orm.go:399]: I've removed it. from here. I just used it to test that errors were properly showing

**Review by samsondav — APPROVED** [review]

**Review by samsondav — APPROVED** [review]

**Pull request merged**